File: Java/Reverse_number.java
package com.lev;

import java.util.Scanner;

public class Reverse_number {
    public static void main(String[] args) {
        System.out.print("Enter number :");
        Scanner a = new Scanner(System.in);
        int n = a.nextInt();
        int res = 0;
        while(n!=0){
            int r = n % 10;
            res = res * 10 + r;
            n = n / 10;
        }
        System.out.print(res);
    }
}

File: C++/Reverse_number.cpp
#include<iostream>
using namespace std;
int main(){
    int n,r,res;
    res=0;
    cout<<"Enter number: ";
    cin>>n;
    while(n!=0){
        r=n%10;
        res=res*10+r;
        n=n/10;
    }
    cout<<res;
    return 0;
}
